modbus_cli/modbus_rtu.py

```diff
@@ -21,8 +22,16 @@ def __init__(self, device, baud, parity, stop_bits, slave_id):
     def connect(self):
         from serial import Serial
 
+        logging.debug("Serial port %s. Parameters: %s baud, %s stop bit(s), parity: %s, timeout %ss.",
```

Comment:
Isn't this maybe to detailed logging, even for debugging? This is information useful for debugging the tool itself, but once the tool it's working, is it really ever useful for the user of the tool?

Reply:
I mean, verbose is _verbose_. 

Anyways, I think it's very nice to have some confirmation that the serial settings you passed are parsed properly, and what they are, particularly when you're starting out with a tool.

Reply:
Well, the registers can come from an env variable, so parsing them can happen behind the scenes and fail in multiple ways so it's nice to have confirmation that it was picked up. The serial parameters on the other hand come straight from the command line, there is no situation where they could be different from the command line.